src/bumper.js

```diff
@@ -97,6 +97,10 @@ class Bumper extends BasePlugin implements IMiddlewareProvider, IAdsControllerPr
    * @returns {void}
    */
   updateConfig(update: Object): void {
+    if (update.id) {
+      // let kaltura player to load the bumper by entryId
```

Comment:
should not add comments of external dependencies here, the plugin doesn't care about kaltura player
Don;t you think this is confusing? either set an ID or a URL, does this handle cases where there are both? what is the flow?

Reply:
the flow is change media from url to id. without this change, kaltura player won't load the bumper by id as url exists